osu.Game.Rulesets.Osu/Mods/OsuModAutopilot.cs

```diff
@@ -32,7 +32,8 @@ public class OsuModAutopilot : Mod, IUpdatableByPlayfield, IApplicableToDrawable
             typeof(ModAutoplay),
             typeof(OsuModMagnetised),
             typeof(OsuModRepel),
-            typeof(ModTouchDevice)
+            typeof(ModTouchDevice),
+            typeof(OsuModBloom)
```

Comment:
Hmm, it's probably fine to allow autopilot?

Reply:
Yeah true no major ideological flaws I suppose. I was airing on the side of incompatible since I thought if people complained we could just remove the incompatibility, but if its fine its fine. Thoughts on the circle size changing mods? (Grow, deflate)

Reply:
No idea.

Reply:
Resolved by [ef9296f](https://github.com/ppy/osu/pull/30299/commits/ef9296f3ada2fe29537373795c3fd98415454193)